meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in

```diff
@@ -104,11 +105,14 @@ boardType=
 wicFile=
 # dev PID and VID
 devID=
+# non-interactive mode
+nonInteractive="n"
 while true; do
 	case "$1" in
 		--wic) wicFile="${2}"; shift 2 ;;
 		--board-type) boardType="${2}"; shift 2 ;;
 		--dev-id) devID="${2}"; shift 2 ;;
+		--yes) nonInteractive="y"; shift 1 ;;
```

Comment:
minor: you can just use `shift` without specifying `1`

Reply:
Fixed.